state/endpoint_bindings.go

```diff
@@ -215,8 +216,10 @@ func (b *Bindings) updateOps(txnRevno int64, newMap map[string]string, newMeta *
 
 	// Make sure that all machines which run units of this application
 	// contain addresses in the spaces we are trying to bind to.
```

Comment:
Please update this comment if we're only validating the previous space of the default endpoint

Reply:
I am not sure what you mean here. We are validating all binding assignments including the application-specific default space ID if the operator is trying to change it to something other than network.DefaultSpaceID.